bundlewrap/utils/ui.py

```diff
@@ -43,7 +43,7 @@ def add_debug_timestamp(f):
     @wraps(f)
     def wrapped(self, msg, **kwargs):
         if self.debug_mode:
-            msg = datetime.now().strftime("[%Y-%m-%d %H:%M:%S.%f] ") + msg
+            msg = "[" + datetime.now().isoformat() + "] " + msg
```

Comment:
While we're micro-optimizing:

```suggestion
            msg = f"[{datetime.now().isoformat()}] {msg}"
```

This is actually faster.